src/frontend/src/test-e2e/misc.test.ts

```diff
@@ -31,6 +31,37 @@ test("Authorize ready message should be sent immediately", async () => {
   });
 }, 300_000);
 
+test("Should allow valid message", async () => {
```

Comment:
added this test because the next test was failing, but had nothing to do with what it was actually testing. This helped clarify what was happening.